content/html/concepts/elements/terms/button/button.md

```diff
@@ -28,7 +28,7 @@ While buttons are often associated with [form](https://www.codecademy.com/resour
 
 Opening and closing `<button>` tags wrap around a block of content. By default, it appears inline, like a [`<span>`](https://www.codecademy.com/resources/docs/html/elements/span) element, and is styled to appear consistent with buttons on the device it is viewed on.
 
-It is possible for the content to be plain text or other HTML [elements](https://www.codecademy.com/resources/docs/html/elements). This can include [lists](https://www.codecademy.com/resources/docs/html/lists), [paragraphs](https://www.codecademy.com/resources/docs/html/paragraphs), or even [images](https://www.codecademy.com/resources/docs/html/images).
+It is possible for the content to be plain text or other HTML [elements](https://www.codecademy.com/resources/docs/html/elements). This can include [lists](https://www.codecademy.com/resources/docs/html/lists), [paragraphs](https://www.codecademy.com/resources/docs/markdown/paragraphs), or even [images](https://www.codecademy.com/resources/docs/html/images).
```

Comment:
```suggestion
It is possible for the content to be plain text or other HTML [elements](https://www.codecademy.com/resources/docs/html/elements). This can include [lists](https://www.codecademy.com/resources/docs/html/lists), [paragraphs](https://www.codecademy.com/resources/docs/html/elements/p), or even [images](https://www.codecademy.com/resources/docs/html/images).
```